src/Keboola/StorageApi/Metadata.php

```diff
@@ -22,85 +22,81 @@ public function __construct(Client $client)
     }
 
     /**
-     * @param $bucketId
+     * @param string $bucketId
      * @return array list of metadata
      */
-    public function listBucketMetadata($bucketId)
+    public function listBucketMetadata($bucketId): array
     {
-        /** @var array $result */
         $result = $this->client->apiGet("buckets/{$bucketId}/metadata");
+        assert(is_array($result));
         return $result;
     }
 
     /**
-     * @param $tableId
+     * @param string $tableId
      * @return array list of metadata
      */
-    public function listTableMetadata($tableId)
+    public function listTableMetadata($tableId): array
     {
-        /** @var array $result */
         $result = $this->client->apiGet("tables/{$tableId}/metadata");
+        assert(is_array($result));
         return $result;
     }
 
     /**
-     * @param $columnId  -- ex: "in.c-bucket.table.column"
+     * @param string $columnId  -- ex: "in.c-bucket.table.column"
      * @return array list of metadata
      */
-    public function listColumnMetadata($columnId)
+    public function listColumnMetadata($columnId): array
     {
-        /** @var array $result */
         $result = $this->client->apiGet("columns/{$columnId}/metadata");
+        assert(is_array($result));
```

Comment:
ty jo nevim jestli bych to delal. to muze dost zmenit chytani vyjimek jinde. Nestaci ten return v anotaci?

Reply:
vyhozeny vyjimky  tuhle preskoci, tady se zpracuje jen realny vystup
```php
// \Keboola\StorageApi\Client::request
        try {
            /**
             * @var ResponseInterface $response
             */
            $response = $this->client->request($method, $url, $requestOptions);
        } catch (RequestException $e) {
            $response = $e->getResponse();
            $body = $response ? json_decode((string) $response->getBody(), true) : [];

            if ($response && $response->getStatusCode() == 503) {
                throw new MaintenanceException(isset($body['reason']) ? $body['reason'] : 'Maintenance', $response && $response->hasHeader('Retry-After') ? (string) $response->getHeader('Retry-After')[0] : null, $body);
            }

            throw new ClientException(
                isset($body['error']) ? $body['error'] : $e->getMessage(),
                $response ? $response->getStatusCode() : $e->getCode(),
                $e,
                isset($body['code']) ? $body['code'] : '',
                $body
            );
        }
```

Reply:
imho OK, je to enom aby phpstan nenadával, klidně by tam mohlo byt enom `/** @var array $result */`